db/schema.rb

```diff
@@ -36,23 +36,6 @@
   add_index "async_transactions", ["transaction_id"], name: "index_async_transactions_on_transaction_id", using: :btree
   add_index "async_transactions", ["user_uuid"], name: "index_async_transactions_on_user_uuid", using: :btree
 
-  create_table "async_transactions", force: :cascade do |t|
```

Comment:
Is the removal of this table just a rebase issue?

Reply:
Looks like it was.